docs/src/main/tut/idl-generation.md

```diff
@@ -201,7 +213,8 @@ import higherkindness.mu.rpc.protocol._
 
 @message case class HelloResponse(reply: String)
 
-@service trait GreeterService[F[_]] {
+@service(Avro) 
```

Comment:
Maybe, to prevent this falls outdated again, would it make sense if this block is compiled with `tut`?